src/orion/testing/algo.py

```diff
@@ -106,6 +112,23 @@ def customized_mutate_example(search_space, rng, old_value, **kwargs):
     return new_value
 
 
+AlgoType = TypeVar("AlgoType", bound=BaseAlgorithm)
+
+
+def create_algo(
```

Comment:
I think this wrapper could be moved to primary_algo.py.

Reply:
Fixed in 52ba2515be09d9ebd5e500dfb10b28a34851f978